src/index.html

```diff
@@ -1,13 +1,291 @@
+    <header class="header">
+      <div class="header__row">
+        <div class="header__top">
+          <a href="#">
+            <img
+              src="img/header/header-logo.png"
+              alt="header-logo"
+              class="header__logo"
+            >
+          </a>
+
+          <div class="header__icons">
+            <a href="tel: +1 234 5555-55-55" class="icon icon--phone">
+              <div class="icon--number">+1 234 5555-55-55</div>
+            </a>
+
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+        </div>
+
+        <div class="header__bottom">
+          <div class="header__title">Take the Streets</div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
+      <div class="container">
+        <div class="menu__row">
+          <div class="menu__top">
+            <a href="#">
+              <img
+                src="img/header/logo-hover.png"
+                alt="logo-hover"
+                class="menu__logo"
+              >
+            </a>
+
+            <div class="menu__icons">
+              <a href="#" class="icon icon--close"></a>
+            </div>
+          </div>
+
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#home">Home</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about">About Us</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#compare">Compare Bikes</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#details">Details</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contacts" >Contacts</a>
+              </li>
+            </ul>
+          </nav>
+
+          <div class="menu__bottom">
+            <div class="menu__number">+1 234 5555-55-55</div>
+            <div class="menu__title">Book a test ride</div>
+          </div>
+        </div>
+      </div>
+    </aside>
+
+    <button onclick="scrollToTop()" id="scrollToTopBtn">Scroll To top</button>
+
+    <main class="content">
+      <div class="container">
+        <div class="content__introduce introduce" id="about">
+          <div class="introduce__title">Move Free</div>
+
+          <div class="introduce__description half">
+              Shift your ride, not gears. Find the fastest way to move in the
+              city as the bike adapts intuitively to power the speed you need.
+          </div>
+        </div>
+
+        <div class="content__goods goods" id="compare">
+          <div class="goods__title">Compare bikes</div>
+
+          <div class="goods__row">
+            <article class="goods__good">
+              <img
+                src="img/content/bicycles/bicycle-first.svg"
+                alt="bicycle-first"
```

Comment:
You now have the same `alt=bicycle-first` attribute for all bikes, the screen reader will read it as three identical images, although it is not. It is better to choose more personalized names: `Bike model Sporty 4`, `Bike model Ride in town` etc.
Fix this in all code.

Reply:
What's your favourite transport ?

Reply:
Probably a car, but for now I ride scooters 😄